TST: doctest failure with v4.0rc1

I see this local failure:

```
======================================================= FAILURES =======================================================
____________________________________________ [doctest] docs/time/index.rst _____________________________________________
1168 of Earth and the Sun from built-in `ERFA <https://github.com/liberfa/erfa>`_ routines,
1169 but one can also use more precise calculations using the JPL ephemerides (which are derived from
1170 dynamical models). An example using the JPL ephemerides is:
1171 
1172 .. doctest-requires:: jplephem
1173 
1174     >>> ltt_bary_jpl = times.light_travel_time(ip_peg, ephemeris='jpl') # doctest: +REMOTE_DATA +IGNORE_OUTPUT
1175     >>> ltt_bary_jpl # doctest: +REMOTE_DATA +FLOAT_CMP
1176     <TimeDelta object: scale='tdb' format='jd' value=[-0.0037715  -0.00377286]>
1177     >>> (ltt_bary_jpl - ltt_bary).to(u.ms) # doctest: +REMOTE_DATA +FLOAT_CMP
Expected:
    <Quantity [0.00058527, 0.00058518] ms>
Got:
    <Quantity [-0.00132325, -0.00132861] ms>

/private/var/folders/dc/hsm7tqpx2d57n7vb3k1l81xw0000gq/T/astropy-test-y9gsqlns/lib/python3.7/site-packages/docs/time/index.rst:1177: DocTestFailure
========================= 1 failed, 14540 passed, 101 skipped, 52 xfailed in 440.81s (0:07:20) =========================
```

Test header:
```
================================================= test session starts ==================================================
platform darwin -- Python 3.7.4, pytest-5.2.1, py-1.8.0, pluggy-0.13.0

Running tests with Astropy version 4.0rc1.
Running tests in astropy docs.

Date: 2019-11-18T19:05:05

Platform: Darwin-17.7.0-x86_64-i386-64bit

Executable: /usr/local/opt/python/bin/python3.7

Full Python Version: 
3.7.4 (default, Jul  9 2019, 18:15:00) 
[Clang 10.0.0 (clang-1000.11.45.5)]

encodings: sys: utf-8, locale: UTF-8, filesystem: utf-8
byteorder: little
float info: dig: 15, mant_dig: 15

Package versions: 
Numpy: 1.17.3
Scipy: 1.3.1
Matplotlib: 3.1.1.post2466+g014aa5b97
h5py: 2.10.0
Pandas: 0.25.3
Cython: 0.29.13
Scikit-image: 0.16.1
asdf: 2.4.2
astropy-helpers: 4.0.rc1

Using Astropy options: remote_data: any.

rootdir: /private/var/folders/dc/hsm7tqpx2d57n7vb3k1l81xw0000gq/T/astropy-test-y9gsqlns/lib/python3.7/site-packages, inifile: setup.cfg
plugins: astropy-header-0.1.1, forked-1.0.2, arraydiff-0.3, dependency-0.4.0, xonsh-0.9.13.dev1, remotedata-0.3.2, openfiles-0.4.0, asdf-2.4.2, doctestplus-0.5.0, cov-2.8.1
collected 14694 items                                                                                                  
```

--- a/docs/time/index.rst
+++ b/docs/time/index.rst
@@ -1175,7 +1175,7 @@ dynamical models). An example using the JPL ephemerides is:
     >>> ltt_bary_jpl # doctest: +REMOTE_DATA +FLOAT_CMP
     <TimeDelta object: scale='tdb' format='jd' value=[-0.0037715  -0.00377286]>
     >>> (ltt_bary_jpl - ltt_bary).to(u.ms) # doctest: +REMOTE_DATA +FLOAT_CMP
-    <Quantity [0.00058527, 0.00058518] ms>
+    <Quantity [-0.00132325, -0.00132861] ms>
 
 The difference between the builtin ephemerides and the JPL ephemerides is normally
 of the order of 1/100th of a millisecond, so the builtin ephemerides should be suitable

